packages-exp/firebase-exp/rollup.config.release.js

```diff
@@ -117,6 +117,43 @@ const appBuilds = [
   }
 ];
 
+const messagingBuilds = [
+  /**
+   * Messaging Browser Builds
+   */
+  {
+    input: 'messaging/index.ts',
+    output: [
+      {
+        file: resolve('messaging', appPkg.main),
+        format: 'cjs',
+        sourcemap: true
+      },
+      {
+        file: resolve('messaging', appPkg.module),
+        format: 'es',
+        sourcemap: true
+      }
+    ],
+    plugins: [...plugins, typescriptPlugin],
+    external
+  },
+
+  /**
+   * Messaging UMD Builds. Required for FM SDK to work but sw does yet support modules.
+   */
+  {
+    input: 'messaging/index.cdn.ts',
+    output: {
+      file: 'firebase-messaging.js',
+      sourcemap: true,
+      format: 'umd',
+      name: `${GLOBAL_NAME}.app`
+    },
+    plugins: [...plugins, typescriptPluginUMD, uglify()]
```

Comment:
See comments on the other rollup file about whether it's intentional to leave out externals, and also `${GLOBAL_NAME}.messaging` and the `globals` property.

Additionally, components in this release rollup file seem to have an `alias` plugin to bundle installations (see line 195), and I think messaging uses installations so it will need that?